models/contacts.js

```diff
@@ -26,34 +28,27 @@ const getContactById = async (contactId) => {
 const removeContact = async (contactId) => {
   try {
     const contacts = await listContacts();
-    const contactToRemove = contacts.find(({ id }) => id === contactId);
-    if (!contactToRemove) {
-      return null;
-    }
-    const newContacts = contacts.filter(({ id }) => id !== contactId);
-    await fs.writeFile(contactsPath, JSON.stringify(newContacts, null, 2), {
-      encoding: "utf-8",
-    });
+    const contactById = await getContactById(contactId);
+    const newContactList = await contacts.filter(({ id }) => id !== contactId);
+    await fs.writeFile(contactsPath, JSON.stringify(newContact, null, 2));
+    return contactById;
   } catch (error) {
     console.log(`Error: ${error.message}`.red);
   }
 };
 
 const addContact = async (body) => {
   try {
-    const { name, email, phone } = body;
-    if (!name || !email || !phone) {
```

Comment:
Sprawdziłaś czy jeśli nie ma jakiegoś klucza to validator który stworzyłaś to obsługuje?

Reply:
nie rozumiem...

Reply:
ok chyba mam